app/metrics.yaml

```diff
@@ -8,6 +8,26 @@ no_lint:
   - CATEGORY_GENERIC
 
 events:
+  app_opened_all_startup:
```

Comment:
We have to change the name to reflect that this is opening intents: `app_received_intent`?

Reply:
done!